tasks/main.yml

```diff
@@ -11,11 +11,81 @@
     path: "{{ certbot_dir }}/certbot-auto"
     mode: 0755
 
+- name: Stop nginx
+  service:
+    name: nginx
+    state: stopped
+  when: certbot_webserver == 'nginx'
+
+- name: Stop apache2
+  service:
+    name: apache2
+    state: stopped
+  when: certbot_webserver == 'apache' and ansible_os_family == 'Debian'
+
+- name: Stop httpd
+  service:
+    name: httpd
+    state: stopped
+  when: certbot_webserver == 'apache' and ansible_os_family == 'RedHat'
+
+- name: Check if letsencrypt certificate file exists
+  stat: path="/etc/letsencrypt/live/{{ certbot_domain }}/privkey.pem"
+  register: r_cert_domain
+
+- name: Run certbot generation for each host
+  command: "{{ certbot_dir }}/certbot-auto certonly --standalone --email {{ certbot_email }} -d {{ certbot_domain }} -d www.{{ certbot_domain }}"
```

Comment:
There are some domains without the `www` (e.g. I have `servercheck.in`—the www redirects to the bare domain); this should probably just use the raw `{{ certbot_domain }}` without the www.